libbeat/processors/add_docker_metadata/watcher.go

```diff
@@ -32,6 +33,12 @@ type Watcher interface {
 
 	// Containers returns the list of known containers
 	Containers() map[string]*Container
+
+	// ListenStart returns a bus listener to receive container started events, with a `container` key holding it
+	ListenStart() bus.Listener
```

Comment:
I wonder if we should put these in the same interface or one more specific to autodiscovery which is composeo f this Watcher interface.